index.html

```diff
@@ -18,6 +18,19 @@
     <title>My Portfolio</title>
```

Comment:
- [ ] [OPTIONAL]: To keep your project professional, you should consider adding a more meaningful title to it. For instance, why not have "_your name's portfolio"? It would look much better!